src/components/details.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import './styles/details.css';
+import { AiOutlineArrowLeft } from 'react-icons/ai';
+
+const Detail = () => {
+  const selectedStockId = useSelector((state) => state.stocks.selectedStock);
+  const stock = useSelector((state) => state.stocks.stocks)
+    .filter((elem) => elem.id === selectedStockId)[0];
+
+  return (
+
+    <div className="flex">
+      <nav className="nav-toggle">
+        <NavLink to="/" className="nav-toggle-back">
+          <AiOutlineArrowLeft style={{ color: 'seashell' }} />
+          2023
+        </NavLink>
```

Comment:
👍 Good job incorporating the top navigation bar on the details page. However, there are a couple of improvements that can help align it more closely with the design:

- [ ] **Icons and Text:** In the design, there are icons and text elements present in the top navigation bar. You've added the left arrow, but there should be additional icons and text as specified in the design. Kindly consider adding these elements to match the design accurately.

- [ ] **Spacing:** You've added a gap before the navigation bar in the top section. However, according to the design, there shouldn't be any extra gap. Please adjust the spacing to match the design's layout.

| Your | Design |
| --- | --- |
| ![image](https://github.com/AbuTalha3/react-csp-metric-app/assets/55840999/09eddb55-932e-4ce9-8438-24831c4aac5f) | ![image](https://github.com/AbuTalha3/react-csp-metric-app/assets/55840999/a2fb6c44-eb36-4c0f-934d-54684b34666b)|


By implementing these suggestions, you'll ensure that the top navigation bar on the details page is more faithful to the design and creates a consistent user experience. Keep up the great work! 👏📌👨‍💻